src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -427,7 +427,7 @@ ExportQueryExecutionTrees::selectQueryResultToStream(
   const auto& idTable = resultTable->idTable();
   // special case : binary export of IdTable
   if constexpr (format == MediaType::octetStream) {
-    for (size_t i : getRowIndices(limitAndOffset, idTable)) {
+    for (size_t i : getRowIndices(limitAndOffset, *resultTable)) {
```

Comment:
In this file there are a lot of `resultTable`s which can be renamed to `result` because the type's name has changed.

Reply:
Doesn't work a lot of the times, because for the json cases the json object is already called `result`